.github/workflows/agent.yaml

```diff
@@ -35,8 +35,8 @@ jobs:
 
       - name: Install R with rig
         run: rig add release
-      - name: Install renv package
-        run: Rscript -e "install.packages('renv')"
+      - name: Install R packages
+        run: Rscript -e "install.packages(c('renv', 'glue'))"
```

Comment:
We needed one extra dependency to check if it exists, glue is low-dependency so should be just fine + quick to install